scalafmt-tests/src/test/resources/test/IndentOperator.stat

```diff
@@ -112,3 +112,25 @@ val x =
 val x = 1 +
   2 +
   3
+<<< #1223 infix with call
+val x = y :: List(
+a,
+b,
+c
+)
+>>>
+val x = y :: List(
+  a,
+  b,
+  c
+)
+<<< #1223 infix operator two
```

Comment:
The following test case passed before this PR and fails after this change
```
<<< ONLY #1223 2 infix operator two
  val x = 1 + 2 +
    3
>>>
val x = 1 + 2 +
  3
```

The failure message
```
[info] - test/IndentOperator.stat: #1223 2 infix operator two                  | *** FAILED ***
[info]   ===========
[info]   => Obtained
[info]   ===========
[info]   val x = 1 + 2 +
[info]   3
[info]
[info]
[info]   =======
[info]   => Diff
[info]   =======
[info]    val x = 1 + 2 +
[info]   -3
[info]   +  3 (FormatTests.scala:81)
```

Reply:
I'm looking at this again, and this behavior might actually be correct/desireable.  I think the infix operators (+, *, /, -) just need to be added to the default exclude list.